tests/hook_main.py

```diff
@@ -2804,15 +2804,11 @@ def discount_delete(transaction):
 @hooks.before("Discount Codes > Get Discount Code Detail using the code > Get Discount Code Detail")
 def discount_code_get_detail_using_code(transaction):
     """
-    GET /discount-codes/DC101
+    GET event_id/1/discount-code/DC101
```

Comment:
And this is absolutely wrong. There is no route `event_id/` in the API. Not sure what you're trying to do

Reply:
@iamareebjamal Had to write `event/`, made a mistake.